com.unity.renderstreaming/Tests/Runtime/RenderStreamingInternalTest.cs

```diff
@@ -143,7 +143,7 @@ public IEnumerator OpenConnectionThrowException(TestMode mode)
 
         [TestCase(TestMode.PublicMode, ExpectedResult = null)]
         [TestCase(TestMode.PrivateMode, ExpectedResult = null)]
-        [UnityTest, Timeout(1000)]
+        [UnityTest, Timeout(10000)]
```

Comment:
extend timeout time
I think sometimes need more time to communication with stun server